PermissionsEx/permissions.yml

```diff
@@ -2302,6 +2302,24 @@ groups:
         - essentials.skull
         - essentials.skull.*
         - essentials.gamemode.survival
+  doctordoom:
+    inheritance:
+    - mortal
+    options:
+      prefix: '&2D&ao&2ct&ao&2r &aD&2o&ao&2m'
+      suffix: '&b'
+    worlds:
+      survival:
+        permissions:
+        - ch.alias.kill
```

Comment:
https://video.search.yahoo.com/yhs/search;_ylt=AwrTcdBiKH1YSyAALxQPxQt.?p=Dr.Doom+exploding+heads+in+Fantastic+Four+movie&fr=yhs-omr-001&fr2=piv-web&hspart=omr&hsimp=yhs-001&type=86311436#action=view&id=5&vid=4af20d00b907867b08585f2441bc282a